tests/Test/Pager/Subscriber/Filtration/Doctrine/ORM/QueryTest.php

```diff
@@ -694,7 +695,43 @@ public function shouldNotFilterWithEmptyParametersAndDefaults(): void
         $this->assertEquals('SELECT a0_.id AS id_0, a0_.title AS title_1, a0_.enabled AS enabled_2 FROM Article a0_ LIMIT 10', $executed[5]);
     }
 
-    protected function getUsedEntityFixtures(): array
+    /**
+     * @test
+     */
+    public function shouldFilterCaseInsensitiveWhenAsked()
+    {
+        $em = $this->getMockSqliteEntityManager();
+        $this->populate($em);
+
+        $dispatcher = new EventDispatcher();
+        $dispatcher->addSubscriber(new PaginationSubscriber());
+        $dispatcher->addSubscriber(new Filtration());
+        $p = new Paginator($dispatcher);
+
+        $this->startQueryLog();
+        $query = $this->em->createQuery('SELECT a FROM Test\Fixture\Entity\Article a');
+        $query->setHint(QuerySubscriber::HINT_FETCH_JOIN_COLLECTION, false);
+        $query->setHint(WhereWalker::HINT_PAGINATOR_FILTER_CASE_INSENSITIVE, true);
+
+        $_GET['filterParam'] = '';
+        $_GET['filterValue'] = 'suMmeR';
+        $defaultFilterFields = 'a.title';
+        $view = $p->paginate($query, 1, 10, compact('defaultFilterFields'));
+        $items = $view->getItems();
+        $this->assertEquals(1, count($items));
+        $this->assertEquals('summer', $items[0]->getTitle());
+
+        $executed = $this->queryAnalyzer->getExecutedQueries();
+
+        // Different aliases separators according to Doctrine version
+        if (version_compare(\Doctrine\ORM\Version::VERSION, '2.5', '<')) {
+            $this->assertEquals('SELECT a0_.id AS id0, a0_.title AS title1, a0_.enabled AS enabled2 FROM Article a0_ WHERE LOWER(a0_.title) LIKE \'summer\' LIMIT 10', $executed[1]);
+        } else {
+            $this->assertEquals('SELECT a0_.id AS id_0, a0_.title AS title_1, a0_.enabled AS enabled_2 FROM Article a0_ WHERE LOWER(a0_.title) LIKE \'summer\' LIMIT 10', $executed[1]);
+        }
+    }
+
+    protected function getUsedEntityFixtures()
```

Comment:
Same as above: missing return type

Reply:
Done.  I've added a native PHP return type of `array` and a PHPDoc comment with the return type of `Article[]`.